Ansible/todoapp-playbook.yml

```diff
@@ -0,0 +1,75 @@
+- name: Run Todo App
+  hosts: webservers
+  remote_user: ec2-user
+
+  vars_prompt:
+  - name: trello_api_key 
+    prompt: Enter your trello api key?
+    private: no
+
+  - name: trello_token
+    prompt: Enter your trello token?
+    private: no
+
+  - name: secret_key
+    prompt: Enter your secret key?
+    private: no 
+
+  tasks:
+  - name: install latest version of git
+    ansible.builtin.yum:
+      name: git
+      state: latest
+    become: yes # become root user, elevating your priviledge to sudo
+
+  - name: install python3
+    ansible.builtin.yum:
+      name: python3
+      state: latest
+    become: yes
+
+  - name: install poetry
+    ansible.builtin.shell:
+      cmd: 'curl -sSL https://install.python-poetry.org | python3 -'
+    become: yes
```

Comment:
See below, but I think becoming root here should be unnecessary - and it's a bad habit to get into to just use root for everything for simplicity! It will make your life easier in the short term, but it leaves bigger security gaps later